lib/presentation/organisms/intro/tips_organism.dart

```diff
@@ -37,6 +41,17 @@ class TipsOrganism extends StatelessWidget {
             translate: false,
           ),
           const SeparatorAtom(variant: SeparatorVariant.farApart),
+          Center(
+            child: Platform.isAndroid
+                ? ButtonAtom(
+                    variant: ButtonVariant.primary,
+                    onPressed: () {
+                      toggleDND(context);
+                    },
```

Comment:
toggleDND does not need context so you can just replace the whole thing with it as a function